ui/src/preview/preview-table-item.tsx

```diff
@@ -61,13 +61,11 @@ export const PreviewTableItem: React.FC<PreviewTableItemProps> = ({
         const filtered = list.filter((e) => e.type === type);
         return filtered.map((error) => t(error.localizationKey, error.params));
     };
-    const errorTexts = issuesToStrings(tableEntry.issues, 'ERROR');
-    const warningTexts = issuesToStrings(tableEntry.issues, 'WARNING');
+
     const hasErrors = tableEntry.issues.length > 0;
 
-    const statusCellClassName = createClassName(
-        hasErrors && styles['preview-table-item__status-cell--expandable'],
-    );
+    const errorTexts = issuesToStrings(tableEntry.issues, 'ERROR');
+    const warningTexts = issuesToStrings(tableEntry.issues, 'WARNING');
```

Comment:
En muista tarvitaanko näitä muuallakin kuin tuolla `<ValidationStateRow>`:ssa, mutta jos ei, niin nämä luonnit ja luontifunkkarit voisi siirtää kanssa sinne